interactions/api/models/gw.py

```diff
@@ -780,9 +780,9 @@ class MessageReaction(DictSerializerMixin):
     emoji: Optional[Emoji] = field(converter=Emoji, default=None)
 
 
-class ReactionRemove(MessageReaction):
+class MessageReactionRemove(MessageReaction):
```

Comment:
Does this still work with all events?

Reply:
I tested all reaction events so yeah thats works